home.css

```diff
@@ -0,0 +1,473 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+body {
+  font-family: 'Lato', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.head {
+  background-image: url("img/background_img.jpg");
+  height: 600px;
+  background-size: cover;
+}
+
+.realHead {
+  display: flex;
+  flex-direction: column;
+}
+
+.sideMenu {
+  display: block;
+  margin-left: 30px;
+  height: 50px;
+  cursor: pointer;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.mobileNav {
+  background-color: white;
+  display: none;
+  height: 500px;
+  margin-top: -20px;
+}
+
+.mobile {
+  list-style-type: none;
+}
+
+.closeMenu {
+  float: right;
+  margin-right: 20px;
+  font-size: 30px;
+  cursor: pointer;
+  color: red;
+}
+
+.navSlide {
+  display: none;
+}
+
+.phrases {
+  margin-top: 20px;
+  margin-left: 30px;
+  color: white;
+}
+
+.phrasesh2 {
+  font-size: 25px;
+}
+
+.phraseh1 {
+  font-size: 34px;
+}
+
+.headParaDiv {
+  margin-left: 30px;
+  border: 1px solid red;
+  background-color: white;
+  opacity: 0.6;
+  margin-right: 30px;
+}
+
+.headPara {
+  margin-left: 5px;
+  color: red;
+}
+
+.headLoc {
+  margin-left: 30px;
+  color: white;
+}
+
+.headL {
+  font-family: "COCOGOOSE", sans-serif;
+}
+
+.headLocw {
+  margin-top: -10px;
+}
+
+.mainDiv {
+  margin-top: -27px;
+}
+
+.wedoB {
+  height: 900px;
+  background-image: url("img/wedo_back.jpg");
+}
+
+.doHead {
+  text-align: center;
+  color: red;
+  font-size: 20px;
+}
+
+.doDivLine {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 10px;
+  margin-top: 60px;
+}
+
+.doDiv {
+  display: flex;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(255, 248, 248, 0.082);
+  margin-left: 10px;
+  margin-right: 10px;
+}
```

Comment:
- [OPTIONAL] You might want to consider using `padding` property to improve the visual design of your page. 😃 

*Yours*
![section_is](https://user-images.githubusercontent.com/79876959/142711015-af4e4259-6d72-4ef6-83cd-252f0c19986b.PNG)

*Suggestion*
![section_suggestion](https://user-images.githubusercontent.com/79876959/142711020-0db662cd-eea2-4e61-b8c9-9abca692ea0e.PNG)